sycl/source/detail/event_impl.hpp

```diff
@@ -278,6 +278,14 @@ class event_impl {
     return MGraph.lock();
   }
 
+  void setEventFromSubmitedExecCommandBuffer(bool value) {
+    EventFromSubmitedExecCommandBuffer = value;
+  }
+
+  bool getEventFromSubmitedExecCommandBuffer() const {
+    return EventFromSubmitedExecCommandBuffer;
+  }
+
```

Comment:
Slight preference for `isEventFromSubmitedExecCommandBuffer` since it makes it clear this is a query for a bool, and is commonly used elsewhere in the runtime.